src/hip_ra_x/hip_ra_x.py

```diff
@@ -186,7 +190,7 @@ def parameter_dict_entry(param: Parameter) -> Parameter:
         )
         self.fluid_heat_capacity: Parameter = parameter_dict_entry(
             floatParameter(
-                'Fluid Heat Capacity',
```

Comment:
See https://github.com/softwareengineerprogrammer/GEOPHIRES-X/pull/7/files for diff scoped to HIP-RA-X